onnx_mxnet/tests/test_models.py

```diff
@@ -33,6 +35,13 @@
     'vgg19_onnx' : 'https://s3.amazonaws.com/download.onnx/models/vgg19.tar.gz'
```

Comment:
it is still reading these files from ONNX repo. Can you change it to read it from ONNX-MXNet repo. So that we have complete control over those files.

Reply:
Yes , we can change that in another PR. Lets fix the build failure first for now.